src/tailwind.css

```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap");
+
```

Comment:
can shift to public index.html instead?

Reply:
ok shifted to public/index.html. but i dont intend to merge this into master though, cause this one should eventually be in tradetrust-ui-components so both creator and tt can use this font.

Reply:
changing to used fonts weights only